checkov/terraform/checks/resource/aws/ElasticacheHasSecurityGroup.py

```diff
@@ -0,0 +1,19 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class ElasticacheHasSecurityGroup(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure no aws_elasticache_security_group resources exist"
+        id = "CKV_AWS_196"
+        supported_resources = ['aws_elasticache_security_group']
+        categories = [CheckCategories.APPLICATION_SECURITY]
```

Comment:
```suggestion
        categories = [CheckCategories.NETWORKING]
```
fits better, we also use it for other similar checks